### ISSUE TITLE
No data at all for 3/14

### ISSUE THREAD

**Issue by marshallstanton** [CSSEGISandData/COVID-19#746]
Confirmed cases are completely blank for all locations/rows.

**Comment by Ruegg** [comment]
This was a very poor decision, it broke my tracker which had 6000 users as of the last 24 hours.

**Comment by Bost** [comment]
:man_facepalming:

**Comment by loulouOz** [comment]
It is been now 3-4 days they have issues, and they fixed it in the next couple of hours. Just condition your app to keep latest data available and that will solve your problems for now even if this is not an ideal solution. 👍

**Comment by Bost** [comment]
> Just condition your app to keep latest data available and that will solve your problems for now even if this is not an ideal solution. +1

The problem is, here in Europe it's almost 2 a.m. since about 4 weeks I go to bed at 4 a.m. every day because all these screw-ups... And I guess I'm not the only one.
And you know Europe is the hotspot now. And the people here - pretty everyone really wants to know what's the progress... bloody hell :(

**Comment by loulouOz** [comment]
I understand, maybe write and automate a little script that will check the last column of the first csv to identify potential errors, if none download all and update your app else wait. You are gonna lose your head if you keep on checking manually... good luck!

**Comment by shahesam84** [comment]
Calm down guys. This happens everyday and they fix it afterwards. If you have problem better to fix your code to avoid showing incomplete data. These guys are doing great providing us with such valuable collected information.

**Comment by Bost** [comment]
> You are gonna lose your head if you keep on checking manually... good luck!

I rely on https://github.com/ExpDev07/coronavirus-tracker-api and they've been delivering on promise (except one exception yesterday - Ok nobody is perfect). So no matter what, I really want to see that my https://t.me/corona_cases_bot is working. Call it a sense of responsibility... Hmm. I'm tired.

**Comment by Ruegg** [comment]
Yeah I fixed mine just by checking whether the last element per line is empty, then removing it.

**Comment by mmonfort2908** [comment]
I'm in Australia and I've noticed that the pattern is they initially update the daily datasets first and then the full time series. The full time series has latest dates added but with blank values and then a while later the values are added. It's not a big deal if you set your app to cope with this. 

Would be great to get some transparency from them on what the schedule but they are doing an amazing service to keep this up. Have any of you guys read the Lancet article they wrote on this that's in the main page of the repo? Looks like there is a large reliance on manual work and people to get this done.

I hope it continues because I'd rather have this and a few delays/hiccups than nothing at all. Let's be encouraging to them please.

**Comment by Bost** [comment]
>  amazing service [..] manual work

The former is unfortunately not true, the later has been obvious since the early days and all in all "amazing service" and "manual work" goes rarely together, when computers are involved.

Computer mouse is the wrong tool for any serious data manipulation and MS Excel is not the data source it's the data sink! :( Don't you see it?

**Comment by yy** [comment]
I'm building an alternative data repository (with code and workflow to clean and process the dataset): https://github.com/covid19-data/covid19-data Feel free to use it (and help)!

**Comment by simra** [comment]
As an alternative for country-level data I'm also pulling the WHO numbers from here:
https://ourworldindata.org/coronavirus-source-data

**Comment by analyzewithpower** [comment]
@Bost  #agree - the data on their map -https://coronavirus.jhu.edu/map.html -is accurate and error-free, why not share that same data with us even if it means restructuring the file?

An export of the data from their dashboard would suffice,

**Comment by analyzewithpower** [comment]
@simra - same here, I am currently looking for a source by state.

**Comment by yy** [comment]
@simra hey I've made a workflow to further clean the Our World in Data team's dataset here: 
 https://github.com/covid19-data/covid19-data#daily-case-data-based-on-who-report-cleaned-by-our-world-in-data

**Comment by Bost** [comment]
> @Bost #agree - the data on their map -https://coronavirus.jhu.edu/map.html -is accurate and error-free, why not share that same data with us even if it means restructuring the file?
> 
> An export of the data from their dashboard would suffice,

That's a very good question! I'm bookmarking myself this conversation!!!

**Comment by mmonfort2908** [comment]
> > amazing service [..] manual work
> 
> The former is unfortunately not true, the later has been obvious since the early days and all in all "amazing service" and "manual work" goes rarely together, when computers are involved.
> 
> Computer mouse is the wrong tool for any serious data manipulation and MS Excel is not the data source it's the data sink! :( Don't you see it?

Better than having nothing out there though right?
